GVFS/GVFS.Notifications/VFSForGit.Mac/StatusMenuItem/IPC/VFSForGitNotification.m

```diff
@@ -0,0 +1,142 @@
+#import "VFSForGitNotification.h"
+
+NSString const * IdentifierKey = @"Id";
+NSString const * EnlistmentKey = @"Enlistment";
+NSString const * EnlistmentCountKey = @"EnlistmentCount";
+NSString const * TitleKey = @"Title";
+NSString const * MessageKey = @"Message";
+
+NSString const * AutomountTitle = @"GVFS AutoMount";
```

Comment:
"NSString * const foo = @"bar";
You won't need to const_cast it to NSString*.

Reply:
Yep... Thank you!